.github/workflows/macos.yml

```diff
@@ -18,17 +18,25 @@ jobs:
       MIX_ENV: test
 
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
 
       - name: Install Erlang and Elixir
         run: |
           brew install erlang elixir
           mix local.hex --force
           mix local.rebar --force
 
-      - name: Compile and Test
+      - name: Compile with elixir_make and Test
         run: |
           mix deps.get
-          mix deps.compile
+          mkdir -p config
+          cat <<EOF | tee config/config.exs
+          import Config
+
+          config :fennec_precompile, :config, stb_image: [
+            fennec_force_build: true,
```

Comment:
Instead of setting `force_build` on the CI file, you can change the version in the mix.exs to `-dev`. :)